templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -708,6 +708,14 @@ Resources:
                     - !Sub "arn:${AWS::Partition}:s3:::${ESBucketName}"
                     - !Sub "arn:${AWS::Partition}:s3:::${ESBucketName}/*"
                 - !Ref "AWS::NoValue"
+        - PolicyName: SSMParameterPutAccess
```

Comment:
Should we create this policy as its own resource then include it in both the app node role and the NFS server role?
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-policy.html

This way it only needs to be defined once